HISTORY.md

```diff
@@ -1,3 +1,8 @@
+3.12.2 / 2018-03-19
```

Comment:
Could you change this from "3.12.2 / 2018-03-19" to "UNRELEASED"?

Reply:
done!